list.py

```diff
@@ -78,33 +81,24 @@ def __getitem__(self, index):
 
 
     def __setitem__(self, index, elem):
-        self.check_index_borders(index)
-        self.find_node(self.index_transform(index)).elem = elem
+        index = self.check_index_and_transform(index)
+        self.find_node(index).elem = elem
         
 
     def __iter__(self):
         return ListIterator(self.head)
 
 
-    def index_transform(self, index: int) -> int:
-        '''
-        Transform negative indices to correct positive value
-        '''
-        return index + self.length if index < 0 else index
-
-
-    def check_index_borders(self, index, border = 1):
+    def check_index_and_transform(self, index, include_border = 0):
```

Comment:
Squashed `def check_index_borders`, `def index_transform` and `def raise_exception_oob` to `def check_index_and_transform`